web_widget_fieldtext_options/README.rst

```diff
@@ -0,0 +1,53 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+=========================
+'Web Text Limited Widget'
+=========================
+
+* Add new 'text_limited' widget for TextField, but data are limited to
+10 lines or 500 characters (by default).
+
+Usage
+=====
+
+* You can change default values by context varibles 'limit_lines' and 'limit_chars'.
+If data contains more characters or lines, it will be cut.
+Example of usage:
+<field name="some_text_field"
+       widget="text_limited"
```

Comment:
Is widget="text_limited" needed? The extension is made with include() and it doesn't registers the widget into instance.web.form.widgets.